src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -366,6 +374,10 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile() {
     result_code = mobile_apis::Result::WRONG_LANGUAGE;
   }
 
+  if (application_manager_.hmi_interfaces().GetInterfaceState(
```

Comment:
@LuxoftAKutsan, 
`GetInterfaceState` return enum, not a `bool`. So? it's not correct to verify it this way.

Could we use something like `IsInterfaceAvalibe(Type):bool`?